UPGRADE.md

```diff
@@ -63,6 +63,8 @@ There is a list of all the repositories maintained by monorepo, changes in log b
     - new macro `loadMoreButton` is integrated into `@ShopsysShop/Front/Inline/Paginator/paginator.html.twig`, update files based on commit from [`ajaxMoreLoader is updated and generalized`](https://github.com/shopsys/shopsys/pull/579/files)
 - *(optional)* [#645 SVG icons in generated document](https://github.com/shopsys/shopsys/pull/645)
     - to display svg icons collection correctly in grunt generated document for all browsers please add `src/Shopsys/ShopBundle/Resources/views/Grunt/htmlDocumentTemplate.html` file and update `src/Shopsys/ShopBundle/Resources/views/Grunt/gruntfile.js.twig` based on changes in this pull request
+- [#660 - Bootstrap: do not prematurely boot Kernel when running in console](https://github.com/shopsys/shopsys/pull/660)
```

Comment:
I think this is optional - the project works without this change

Reply:
I was thinking about it for a bit when I wrote that upgrade note - yes, it works, but #658 is tagged as a bug and I don't feel good about marking bugfixes optional. Are you sure we should mark this change as optional?

Reply:
Ok... so we have one idea why it should be optional and one why it shouldn't.
For a project that was created before #596 this is optional, for project that was created after #596, this is required.

So again - two ideas against two ideas... 

After some consideration, I don't know if there should be optional or not. If you think there shouldn't be, leave it as it is :+1:

Reply:
You are right! It's tied to whether you have #596 merged in your repo. Fortunately it's merged only in `master`, so we can append note about this to the original optional upgrade note.

I've done that in 43fe6b307 - what do you think? I think it might be the best solution, as this fixes a bug introduced in the #596.

Reply:
Nice solution. I would prefer `[the PR #660]` over ~`[the PR #660 diff]`~

Reply:
Done :slightly_smiling_face: Can I merge?

Reply:
I cannot approve it more :smiling_imp: